modules/web-console/e2e/testcafe/mocks/WebSocketHook.js

```diff
@@ -26,23 +26,40 @@ export class WebSocket {
 
     /**
      * @param {string} eventType
+     * @param {string} goalEventType
      * @param {(any)=>any} listener
      */
-    on(eventType, listener) {
+    _on(eventType, goalEventType, listener) {
         this.socket.on('message', (msg) => {
             const e = JSON.parse(msg);
 
             if (e.eventType === eventType) {
                 const res = listener(e.payload);
 
                 if (res)
-                    this.emit(e.eventType, res, e.requestId);
+                    this.emit(goalEventType, res, e.requestId);
             }
         });
 
         return this;
     }
 
+    /**
+     * @param {string} eventType
+     * @param {(any)=>any} listener
```

Comment:
```suggestion
     * @param {(ws: WebSocket) => void} listener
```
Same for other hook annotations. If you choose to update types in other files too, don't forget to use import syntax, i.e. `import('../WebSocketHook').WebSocket`.

Reply:
Not required. Argument type is not WebSocket.